iogt/styles/iogt/state_320/_s-menu.scss

```diff
@@ -20,14 +23,13 @@
 }
 .header__nav-item {
   text-decoration: none;
-  &:last-child {
-    padding-right: 0;
-  }
+  padding-left: 10px;
```

Comment:
This could be on one line?

Reply:
I didn't really wanna mess with any top or bottom padding, but there's no inherited values, so it works fine with the 0 values in the shorthand 😎thanks